### ISSUE TITLE
[BUG] [Formatter] [nunjucks] v1.28.0 ignores  djlint:off when a macro is called

### ISSUE THREAD

**Issue by shaileshaanand** [Riverside-Healthcare/djLint#659]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: alpine 3.18 (Docker: python:3.11.3-alpine3.18)
 - Python Version 3.11.3
 - djLint Version 1.28.0
 - template language: nunjucks

## Issue

djlint 1.28.0 ignores the djlint:off and djlint:on tags when a macro is being called.
This issue started from v1.28.0

## How To Reproduce

Setup:
test.njk
```html
<ul>
  {# djlint:off #}
  <li>{{foo(1)}}</li>
  {# djlint:on #}
</ul>
```

Expected Behavior: (taken from v1.17.2)

```
djlint --reformat --profile nunjucks ./test.njk
###
Reformatting 1/1 files ━━━━━━━━━━━━━━━━━━━━ 00:00    


test.njk
──────
@@ -1,5 +1,5 @@

 <ul>
-  {# djlint:off #}
+    {# djlint:off #}
   <li>{{foo(1)}}</li>
-  {# djlint:on #}
+    {# djlint:on #}
 </ul>

1 file was updated.
```

Actual Behavior:
```
djlint --reformat --profile nunjucks ./test.njk
###
Reformatting 1/1 files ━━━━━━━━━━━━━━━━━━━━━ 00:00    


test.njk
──────
@@ -1,5 +1,5 @@

 <ul>
-  {# djlint:off #}
-  <li>{{foo(1)}}</li>
-  {# djlint:on #}
+    {# djlint:off #}
+  <li>{{ foo(1) }}</li>
+    {# djlint:on #}
 </ul>

1 file was updated.
```
